docs/developer-guide/syntax-reference.rst

```diff
@@ -379,10 +379,7 @@ timestamp and message key, respectively. The timestamp has milliseconds accuracy
 
 KSQL has currently the following requirements for creating a table from a Kafka topic:
 
-1. The Kafka message key must also be present as a field/column in the Kafka message value. The ``KEY`` property (see
-   below) must be defined to inform KSQL which field/column in the message value represents the key. If the message key
-   is not present in the message value, follow the instructions in :ref:`ksql_key_requirements`.
-2. The message key must be in ``VARCHAR`` aka ``STRING`` format. If the message key is not in this format, follow the
+1. The message key must be in ``VARCHAR`` aka ``STRING`` format. If the message key is not in this format, follow the
```

Comment:
a list with one element? I think we should simplify this and merge it with the above line